__tests__/app.test.js

```diff
@@ -289,6 +280,62 @@ describe("Northcoders News API", () => {
 
   })
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("Status: 200 - Should respond with an array of the comments for the given article id of 1 which has 11 comments.", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ _body: comments }) => {
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments).toHaveLength(11);
+        });
+    });
+
+    it("Status: 200 - Should respond with an array of comments that each include the following properties: comment_id, votes, created_at, author and body", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ _body: comments }) => {
+          comments.forEach((comment) => {
+            expect(comment).toEqual(
+              expect.objectContaining({
+                comment_id: expect.any(Number),
+                votes: expect.any(Number),
+                created_at: expect.any(String),
+                author: expect.any(String),
+                body: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+
+    it("Status: 200 - Should respond with the comments for article_id 1 that is sorted in descending order with the most recent comments at the top", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .then(({ _body: comments }) => {
+          expect(comments).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+
+    it("Status: 400 - Should respond with an error of 400 & message saying that the article id is an invalid data type", () => {
+      return request(app)
+        .get("/api/articles/not-a-number/comments")
+        .expect(400)
+        .then(({ text: msg }) => {
+          expect(msg).toBe("Invalid Data Type");
+        });
+    });
+
+    it("Status: 404 - Should respond with an error of 404 & message saying that no comments not found were found", () => {
+      return request(app)
+        .get("/api/articles/2/comments")
+        .then(({ text: msg }) => {
+          expect(msg).toBe("No comments found");
+        });
+    });
+
```

Comment:
it would be nice to have a test for if an article exists but has no comments (status 200)

Reply:
> it would be nice to have a test for if an article exists but has no comments (status 200)

Thank you for the code review Heather! 😁 I was thinking about how to get the amount of articles currently available dynamically rather than hardcoding it and found out I can do a query to the db, store the article ids into an array and then use that for a conditional later down the line to check if the article id passed in is greater than the amount of articles in the existing articles array 👏👏